skyportal/app_server.py

```diff
@@ -427,7 +427,7 @@ def log_request(self, handler):
         r'/api/webhook/(obj)_analysis/([0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12})?',
         AnalysisWebhookHandler,
     ),
-    (r'/api/internal/tokens(/.*)?', TokenHandler),
+    (r'/api/internal/tokens(/[0-9A-Za-z-_\.\+]+)?', TokenHandler),
```

Comment:
Are you sure you need the dash before the underscore?

Reply:
Yeah its broken without it.

Reply:
ok, just making sure

Reply:
So we limit the token characters to `0-9`, `A-Z`, `a-z`, `-`, `_`, `.`, and `+`. I didn't know we ever needed plusses or underscores?

Reply:
I was just following the convention of the IDs, and that seemed to work ok. Users can name their tokens as they like, so I assume we need to allow the flexibility.

Reply:
Odd, I thought we always generate the IDs ourselves (even though users can name the tokens)?  If the users can indeed choose the name that appears in the URL, we should probably add a hint to the frontend of what they are allowed to enter.

Reply:
Ah I see what you mean, because the ID and the name is different. Ok I take that back, so we can restrict the input somehow.

Reply:
I suspect we use UUID4, so  `0-9a-e-` should work.